packages/stream-tests/src/__tests__/multiquery.test.ts

```diff
@@ -54,66 +54,6 @@ describe('multiquery API http-client tests', () => {
     expect(Math.abs(expectedTimestamp - givenTimestamp)).toBeLessThan(5)
   }
 
-  it('loads the same stream at multiple points in time using atTime', async () => {
```

Comment:
Shouldn't we just change this test to use `opts.atTime`? Would be good to know that this still works as 3ID relies on being able to query the same stream at multiple timestamps.

Reply:
there is already a tests for this right under this test. It is called "loads the same stream at multiple points in time using opts.atTime"

Reply:
Ah ok, my bad!